test/regress/regress1/sets/finite-type/sets-card-neg-mem-union-1.smt2

```diff
@@ -0,0 +1,30 @@
+(set-logic QF_ALL)
+;(set-info :status unsat)
```

Comment:
This seems to be the same file as test/regress/regress1/sets/finite-type/sets-card-neg-mem-union-2.smt2 except for this commented out line and this file not appearing in the list of regressions, so we can probably remove this file from the PR.

Reply:
Yes. It is fixed now.